Lib/Utils/Notification/NotificationManager.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Diagnostics;
+
+namespace Lib.Utils.Notification
+{
+    public class NotificationManager
+    {
+        public void SendNotification(int errors, int warnings, double time)
+        {
+            void SendWindowsNotification()
+            {
+                const string SendNotificationPath = @"Resources\SendNotification.ps1";
+                var p = Process.Start("Powershell.exe", $"{SendNotificationPath} -Errors {errors} -Warnings {warnings} -Time {time}");
```

Comment:
How this can work when current directory is not where bb.exe is?

Reply:
You are right, fixed.